### PR TITLE
Update influx.md

### PR DESCRIPTION
**Pull request by tmactao**
Update the SQL statement in the rule configuration file influxRule.txt

### CODE REVIEW

File: docs/zh_CN/plugins/sinks/influx.md [hunk 1]
```diff
@@ -39,7 +39,7 @@
 ```json
 {
   "id": "influx",
-  "sql": "SELECT * from  demo_stream where temperature < 50",
+  "sql": "SELECT * from  demo_stream where temperature > 50",
```
Comment (ngjaying) [inline docs/zh_CN/plugins/sinks/influx.md:42]: Could you please explain why making this change?

Comment (tmactao) [reply docs/zh_CN/plugins/sinks/influx.md:42]: You can check the conditions in the example usage, the configuration file (influxRule.txt) is "an example for selecting a temperature greater than 50 degrees", so I change it

Comment (ngjaying) [reply docs/zh_CN/plugins/sinks/influx.md:42]: OK，thank you. Could you please also change the same file in the en_US? Thanks

Comment (tmactao) [reply docs/zh_CN/plugins/sinks/influx.md:42]: Ok, I will modify it

**Review by ngjaying — APPROVED** [review]

**Pull request merged**